CONTRIBUTING.md

```diff
@@ -2,61 +2,159 @@
 
 :+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
 
-Following you will see some guidelines about how to contribute with
-JavaScript SDK.
+We welcome contributions from the community! Please take some time to become
+acquainted with the process before submitting a pull request.
 
-## Branch Management
+## Pull Requests
 
-We use Gitflow to manage our branches and that's ok when `develop` branch is
-ahead of `master`.
+When creating a pull request, first fork this repository and clone it to your
+local development environment. Then add this repository as the upstream.
 
-- [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/) by @nvie
+```console
+git clone https://github.com/mygithuborg/sdk-javascript.git
+git remote add upstream https://github.com/cloudevents/sdk-javascript.git
+```
 
-## Changelog
+Typically a pull request should relate to an existing issue. If you have
+found a bug, want to add an improvement, or suggest an API change, please
+create an issue before proceeding with a pull request. For very minor changes
+such as typos in the documentation this isn't really necessary.
+
+Create a branch for your work. If you are submitting a pull request that
+fixes or relates to an existing GitHub issue, you can use this in your
+branch name to keep things organized. For example, if you were to create
+a pull request to fix
+[this error with `httpAgent`](https://github.com/cloudevents/sdk-javascript/issues/48)
+you might create a branch named `48-fix-http-agent-error`.
+
+```console
+git fetch upstream
+git reset --hard upstream/master
+git co -b 48-fix-http-agent-error
```

Comment:
Minor change to use `checkout` instead of `co`.

I don't have the co alias so it gets confusing:
```
09:07 $ git co
git: 'co' is not a git command. See 'git --help'.
```